### PR TITLE
postmessage: Fix show hide / button / command in Notebookbar

### PR DESCRIPTION
**Pull request by hfiguiere**
This is a regression from commit 9923e695959
This is because jQuery .child() is only for the direct descendants. Use .find() instead which will search down the DOM tree.

This restore postmessage Hide_Command / Hide_Button and Show.

Change-Id: Ibe84c520c9786c6c0026d3507a27a8d3580b38a0

* Resolves: # 
* Target version: master (24.04)

### Summary

This is in 24.04 only

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [ ] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/control/Control.Notebookbar.js [hunk 1]
```diff
@@ -336,22 +336,22 @@ L.Control.Notebookbar = L.Control.extend({
 	},
 
 	showNotebookbarButton: function(buttonId, show) {
-		var button = $(this.container).children('#' + buttonId);
+		var button = $(this.container).find('#' + buttonId);
```
Comment (hfiguiere) [inline browser/src/control/Control.Notebookbar.js:339]: the selector before was using the "descendant" combinator ` `.

**Comment by hfiguiere** [comment]
The test failure is an assertion in coolwsd. Unrelated.

**Review by gokaysatir — APPROVED** [review]
Nice catch, thanks :) We could also use this.container.querySelector("...") i guess.

**Review by gokaysatir — APPROVED** [review]

**Review by gokaysatir — APPROVED** [review]

**Comment by hfiguiere** [comment]
> Nice catch, thanks :) We could also use this.container.querySelector("...") i guess.

or `.getElementById` in that case.

**Pull request merged**